File: src/main/java/model/FruitName.java
package model;

public enum FruitName{
    PEACH("Peach"),
    CHERRY("Cherry"),
    PEAR("pear");


    private final String type;

    FruitName(String type) {
        this.type = type;
    }

    public String getFruitType() {
        return type;
    }
}

File: src/main/java/model/FruitName.kt
package model

enum class FruitName {

    var PEACH: FruitName? = null,
    var CHERRY: FruitName? = null,
    var PEAR: FruitName? = null


    private val type: String? = null

    open fun FruitName(type: String?) {
        this.type = type
    }

    open fun getFruitType(): String? {
        return type
    }
}
